packages/esm-tb-app/src/views/treatment-and-follow-up/tb-treatment-follow-up.component.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { EmptyStateComingSoon } from '@ohri/openmrs-esm-ohri-commons-lib';
+
+interface TbTreatmentFollowUpListProps {
+  patientUuid: string;
+}
```

Comment:
Reuse the interface defined in the commons. I think it's `PatientChartProps` to avoid repetition